src/app/components/HelloWorld/HelloWorld.ts

```diff
@@ -0,0 +1,6 @@
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component
+export default class HelloWorld extends Vue {
+  @Prop() private msg!: string;
```

Comment:
Personally, I prefer longer and readable property names like `message` instead of `msg`

Reply:
This example will be removed on a later time.